server/page.js

```diff
@@ -1,10 +1,36 @@
-function page (contents, dehydratedAppState) {
+const xtend = require('extend')
+const serializeJS = require('serialize-javascript')
+
+function page (route, contents, appState) {
```

Comment:
i would change this from `route` to `url`, since in choo terminology `route` means something slightly different that what we are passing in here.